test/modtoken/modtoken.cpp

```diff
@@ -68,4 +71,26 @@ TEST_CASE("NMODL Lexer returning valid ModToken object", "[token][modtoken]") {
     }
 }
 
+TEST_CASE("Addition of two ModToken objects", "[token][modtoken]") {
+    SECTION("adding two random strings") {
+        ast::Name value;
+        {
+            std::stringstream ss;
+            symbol_type("text", value);
+            ast::Name value1 = value;
+            ss << *(value.get_token());
+            ss << " + ";
+            symbol_type("\ntext", value);
+            ast::Name value2 = value;
+            ss << *(value.get_token());
+            ModToken sum = *(value1.get_token()) + *(value2.get_token());
+            ss << " = " << sum;
+            REQUIRE(ss.str() ==
```

Comment:
@iomaganaris : for this unit test, can't we just instantiate ModToken objects and sum them? (i.e. without going through lexer to get tokens)

Reply:
You are right. This is possible and now done :+1: